src/lib/mailerliteClient.ts

```diff
@@ -0,0 +1,9 @@
+import MailerLite from '@mailerlite/mailerlite-nodejs'
+
+export const mailerLiteClient = new MailerLite({
+  api_key: process.env.NEXT_PUBLIC_MAILERLITE_KEY || '',
+})
+
+export const devGroup = '111547703431792014'
```

Comment:
**nit:** this should be an enum as you won't know how many other consts are available, as it will grow, so:

```ts
enum MailerLiteGroups {
  devs = '111547703431792014',
  lookingForDevs = '111547722557818681',
}
```